requirements/base.txt

```diff
@@ -8,7 +8,7 @@ PyYAML~=6.0,>=6.0.1
 cookiecutter~=2.5.0
 aws-sam-translator==1.82.0
 #docker minor version updates can include breaking changes. Auto update micro version only.
-docker~=6.1.0
+docker~=7.0.0
```

Comment:
Ran dev canaries on this PR and all the tests are passing on Linux and Windows:
- https://ci.appveyor.com/project/AWSSAMCLI/aws-sam-cli-canary-linux-dev/builds/48760782
- https://ci.appveyor.com/project/AWSSAMCLI/aws-sam-cli-canary-windows-dev/builds/48760790